plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -208,6 +210,12 @@ public static function get_json_schema(): array {
 					'required'    => true,
 					'readonly'    => true, // Omit from REST API.
 				),
+				'eTag'      => array(
```

Comment:
I wonder if we should make this just `'etag'`. In HTTP it is "ETag", and a lower case "eTag" feels like it would maybe be something else, like an "electronic tag" or something.

Reply:
Makes sense. Done in 08f8f8a.